base/src/iosMain/kotlin/KalugaThread.kt

```diff
@@ -39,5 +41,13 @@ actual data class KalugaThread(val thread: NSThread) {
             "Thread ${(threadDescription["number"] ?: threadDescription["num"]).orEmpty().ifEmpty { "Unknown" }}"
         }
     }
+        set(value) { thread.name = value }
+    actual var priority: Int
+        get() = (thread.threadPriority * MAX_PRIORITY + MIN_PRIORITY).toInt()
+        set(value) {
+            require(value in MIN_PRIORITY..MAX_PRIORITY)
+            thread.setThreadPriority((value - MIN_PRIORITY).toDouble() / MAX_PRIORITY)
```

Comment:
There should be a test for this.

Reply:
the thing is that an expected class can't have a companion object field that is not expected... maybe I'll make it an extension property on a companion..

Reply:
done

Reply:
well it seems like there are some security implications.. scraped the api for now as it cause more problems than it is useful